libs/vscode/nx-workspace/src/lib/get-pdv-data.ts

```diff
@@ -0,0 +1,9 @@
+import { NxPDVDataRequest } from '@nx-console/language-server/types';
+import { PDVData } from '@nx-console/shared/types';
+import { sendRequest } from '@nx-console/vscode/lsp-client';
+
+export async function getPDVData(
```

Comment:
This is a pattern across Nx Console and I don't want to break it in this PR. 
You definitely have a point though, there isn't a huge amount of value gained from this specific function. However, some of the other pdv request functions have different signatures or include some sort of mapping of inputs or outputs so there is more value to them. 
I think it isn't very harmful to have this layer between things so that the vscode extensions doesn't have to concern itself with where the data come from.